test/integration/stackstorm/controls/services_test.rb

```diff
@@ -27,14 +27,14 @@
   # st2auth
   describe port(9100) do
     it { should be_listening }
-    its('addresses') { should include '127.0.0.1' }
```

Comment:
These services should still listen on 127.0.0.1, not on 0.0.0.0.

Reply:
Reverted the tests to check for 127.0.0.1, and changed services to bind to that IP as well